dockernet/src/start_relayers.sh

```diff
@@ -21,11 +21,11 @@ for chain in ${HOST_CHAINS[@]}; do
     printf "STRIDE <> $chain - Adding relayer keys..."
     $relayer_exec rly keys restore stride $RELAYER_STRIDE_ACCT "$mnemonic" >> $relayer_logs 2>&1
     $relayer_exec rly keys restore $chain_name $account_name "$mnemonic" >> $relayer_logs 2>&1
-    echo "Done restoring relayer keys"
+    echo "Done"
```

Comment:
why remove these lines?

Reply:
It's super nit haha, but this log is printed on one line so the extra description winds up being a little redundant / less clean
```
STRIDE <> GAIA - Adding relayer keys...Done restoring relayer keys
STRIDE <> GAIA - Creating client, connection, and transfer channel...Done creating client, connection, and transfer channel
vs
STRIDE <> GAIA - Adding relayer keys...Done 
STRIDE <> GAIA - Creating client, connection, and transfer channel...Done
```